sys/net/network_layer/ng_ipv6/netif/ng_ipv6_netif.c

```diff
@@ -444,12 +449,12 @@ static ng_ipv6_addr_t *_match_prefix(kernel_pid_t pid, const ng_ipv6_addr_t *add
 ng_ipv6_addr_t *ng_ipv6_netif_match_prefix(kernel_pid_t pid,
         const ng_ipv6_addr_t *prefix)
 {
-    return _match_prefix(pid, prefix, false);
+    return _match_prefix(pid, prefix, 0x00);
 }
 
 ng_ipv6_addr_t *ng_ipv6_netif_find_best_src_addr(kernel_pid_t pid, const ng_ipv6_addr_t *dest)
 {
-    return _match_prefix(pid, dest, true);
+    return _match_prefix(pid, dest, _FLAG_ONLY_UNICAST | _FLAG_ONLY_SUBNET);
```

Comment:
I'm really sorry but I think I fail to understand how the fix works if ``_FLAG_ONLY_SUBNET `` seems to be always set by default.. :(